plugins/services/src/js/service-configuration/PodStorageConfigSection.js

```diff
@@ -26,11 +25,11 @@ class PodStorageConfigSection extends React.Component {
         heading: 'Read Only',
         prop: 'readOnly',
         render(prop, row) {
-          return (
-            <ConfigurationMapBooleanValue
-              options={BOOLEAN_OPTIONS}
-              value={row[prop]} />
-          );
+          if (row[prop]) {
+            return BOOLEAN_OPTIONS.truthy;
```

Comment:
just making sure we are expecting to send the `TRUE` and `FALSE` as string and not booleans.